### PR TITLE
Add templates for issues and PRs

### PR DESCRIPTION
**Pull request by adthrasher**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (mcrusch) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Typically, people will not execute this on their desktop.  I'd recommend changing this to "Environment" or "System Information" or something like that, and specify that this is the environment where it was run.  Note that the template should probably also be updated, as most tools will have the same problem.

**Review by mcrusch — CHANGES_REQUESTED** [review]
I think the Desktop thing should be fixed, but looks good otherwise.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (adthrasher) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Agreed, I switched to "Environment". I'll also suggest that Jordan make this change in the template.

**Comment by adthrasher** [comment]
@mcrusch - I switched to a generic "Environment" section. The environment subsection was replaced with "Runner" as an option to indicate running in Docker vs. native.

**Pull request merged**